### DISCUSSION TITLE
NBD, but if you want a clean port to NextJS 15.0.3

### DISCUSSION THREAD

**Discussion by sid-newby** [assafelovic/gpt-researcher#995] [Show and tell]
Along with it a few fixes like a default config, (current ver cant find it) a few bug fixes, I dont normally contribute much because ive got life issues and an unpredictable work schedule but i'd be happy to share. added a handler for bad image links (current ver will push a url with no img extension to the frontend resulting in a broken image thumbnail), a bunch of stuff really that might clear out some backlog.  Just let me know and ill get into a fork, or I can add you dudes to the current project can just download it from my repo.

**Comment by ElishaKay** [comment]
Welcome @sid-newby 

A PR on this repo would be highly appreciated
looking forward 😊
